packages/candy_editor/src/constants.ts

```diff
@@ -26,4 +27,5 @@ export const VALIDATION_ERRORS: { [key: string]: string } = {
   INT16: 'The value must be a Int number between -32768 and 32767',
   INT32: 'The value must be a Int number between -2147483648 and 2147483647',
   INT64: 'The value must be a Int number between -9223372036854775808 and 9223372036854775807',
+  BOOLEAN: 'The value must be a Boolean (true or false - case insensitive)',
```

Comment:
These should not be uppercase. Only the object name VALIDATION_ERRORS needs to represent that the property values are constants.

Reply:
Done, removed the period in Nat as well